components/overflow-group/overflow-group.js

```diff
@@ -249,8 +248,8 @@ class OverflowGroup extends RtlMixin(LocalizeCoreElement(LitElement)) {
 
 		this._getItems();
 
-		this.resizeObserver = new ResizeObserver(this._throttledResize);
-		this.resizeObserver.observe(this._container);
+		this._resizeObserver = new ResizeObserver(this._throttledResize);
+		this._resizeObserver.observe(this._container);
```

Comment:
If `this._resizeObserver` is never referenced outside of here, could it just be a local variable?